pkg/cloud-provider/cloudapi/aws/aws_ec2.go

```diff
@@ -165,6 +170,21 @@ func (ec2Cfg *ec2ServiceConfig) getCachedVpcNameToID() map[string]string {
 	return vpcNameToIDCopy
 }
 
+// GetCachedVpcs returns VPCs from cached snapshot for the account.
+func (ec2Cfg *ec2ServiceConfig) GetCachedVpcs() []*ec2.Vpc {
+	snapshot := ec2Cfg.resourcesCache.GetSnapshot()
+	if snapshot == nil {
+		awsPluginLogger().V(4).Info("cache snapshot nil", "service", awsComputeServiceNameEC2,
```

Comment:
is it error case? considering making it info, as customer onboarding cloud account will have 1 VPC

Reply:
Done